app/controllers/concerns/organization_webhook.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+module OrganizationWebhook
+  extend ActiveSupport::Concern
+
+  def create_organization_webhook
```

Comment:
Rubocop will complain if we put this method inside `OrganizationController`. 

I'm not sure if it's correct to extract this as a `concern`.